examples/nD_image.py

```diff
@@ -4,14 +4,13 @@
 
 import numpy as np
 from skimage import data
-from skimage.color import rgb2gray
-from napari import Window, Viewer
+from napari import ViewerApp
 from napari.util import app_context
 
 
 with app_context():
-    # create the viewer and window
-    viewer = Viewer()
-    window = Window(viewer)
-    # add the image
-    viewer.add_image(np.random.rand(500, 500, 20, 10))
+    # TODO: change axis back to 0 once dims displayed in sane order
+    blobs = np.stack([data.binary_blobs(length=128, blob_size_fraction=0.05,
+                                        n_dim=3, volume_fraction=f)
+                     for f in np.linspace(0.05, 0.5, 10)], axis=-1)
+    viewer = ViewerApp(blobs)
```

Comment:
I get a whole bunch of errors / warnings with this example now when I move the mouse over the image or adjust the clims

```python
  File "/Users/nicholassofroniew/Github/napari/napari/layers/_image_layer/model.py", line 387, in get_value
    msg = msg + f'{value:0.3}'
ValueError: Precision not allowed in integer format specifier
ERROR: Invoking <bound method QtViewer.on_mouse_move of <napari.components._viewer.view.main.QtViewer object at 0x111b10ca8>> for MouseEvent
WARNING: Traceback (most recent call last):
  File "/Users/nicholassofroniew/Github/napari/napari/layers/_image_layer/view/controls.py", line 33, in clim_slider_changed
    cmin = valmin+slidermin*(valmax-valmin)
TypeError: numpy boolean subtract, the `-` operator, is deprecated, use the bitwise_xor, the `^` operator, or the logical_xor function instead.
```

Not clear to me that the blobs are more visually appealing / clearer to look at then the random noise. Same is true for the `nD_markers.py` example.

Reply:
> Not clear to me that the blobs are more visually appealing / clearer to look at

Huh? it's night and day for me, you get some visual continuity between the 3D planes!

Anyway, it seems we can't display boolean images, easy enough fix.

Reply:
Fair enough, thanks for the fix. We can work on the data for examples in future PRs!